book/listings/interface_builder/1/window.ui

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <object class="GtkApplicationWindow" id="window">
+    <property name="title">My GTK App</property>
+    <child>
+      <object class="GtkButton" id ="button">
+        <property name="label">Press me!</property>
+        <property name="margin_top">12</property>
```

Comment:
properties should be written with a `-` instead of `_`. It's a bad habit from glade.